### PR TITLE
jekyll: add page

### PR DESCRIPTION
**Pull request by RioSatria**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/common/jekyll.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Jekyll
+
+> Jekyll is a simple, blog-aware, static site generator.
```
Comment (sbrl) [inline pages/common/jekyll.md]: I'm not sure that we need to say "Jekyll" in both the title and description. Perhaps we could remove the `Jekyll is` from the beginning?

File: pages/common/jekyll.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Generate a development server that will run at http://localhost:4000/:
+
+`jekyll serve`
+
+- Enable incremental regeneration:
+
+`jekyll serve --incremental`
+
+- Generated the current folder into ./_site:
```
Comment (sbrl) [inline pages/common/jekyll.md]: Perhaps `Generated` -> `Generate`? Also, backticks around `./_site` would help neaten things up :-)

File: pages/common/jekyll.md (hunk 1 above)
Comment (RioSatria) [reply pages/common/jekyll.md]: Good idea, i'll remove the `Jekyll is`.

File: pages/common/jekyll.md (hunk 2 above)
Comment (RioSatria) [reply pages/common/jekyll.md]: Thank you for the input. I've corrected my PR.

**Comment by sbrl** [comment]
Thanks, @RioSatria :smiley_cat:

**Comment by agnivade** [comment]
Jekyll should be in lower case. And there seems to be some small backtick and quotes issues. I will file a PR separately.

**Comment by sbrl** [comment]
@agnivade Oops! I'll have to be more careful in the future.

**Pull request merged**